support/mesh/mesh_protobuf/src/message.rs

```diff
@@ -130,10 +133,13 @@ impl ProtobufAny {
 
 #[cfg(test)]
 mod tests {
+    extern crate std;
```

Comment:
Is this actually necessary? IIRC, tests will always pull in std, no?

Reply:
`#[no_std]` still means `std` is not implicitly available, even for tests.